src/script/api/script_date.hpp

```diff
@@ -48,69 +189,145 @@
  *     Economy-time will use a 360 day calendar (12 months with 30 days each), which runs at a constant speed of one economy-month per realtime-minute.
  *     Calendar-time will use a Gregorian calendar, which can be slowed to stopped via game settings.
  */
-class ScriptDate : public ScriptObject {
+class ScriptEconomyDate : public ScriptObject {
 public:
 	/**
-	 * Date data type is an integer value. Use ScriptDate::GetDate to
-	 * compose valid date values for a known year, month and day.
-	 */
+	* Date data type is an integer value. Use ScriptEconomyDate::GetDate to
+	* compose valid date values for a known year, month and day.
+	*/
```

Comment:
Spaces went missing in this file before '*'s of comments. Both for `ScriptEconomyDate` and `ScriptCalendarDate`.

Reply:
Ha yes copy/paste tends to do that, will fix.